src/term/term_to_net.rs

```diff
@@ -1,17 +1,50 @@
 use super::{DefId, DefinitionBook, Name, Op, Term};
 use crate::net::{INet, NodeId, NodeKind::*, Port, LABEL_MASK, ROOT};
-use hvmc::run::Val;
+use bimap::BiHashMap;
+use hvmc::{
+  ast::{name_to_val, val_to_name},
+  run::Val,
+};
 use std::collections::HashMap;
 
-pub fn book_to_compact_nets(book: &DefinitionBook) -> anyhow::Result<Vec<(DefId, INet)>> {
-  let mut nets = Vec::new();
+pub fn book_to_compact_nets(
```

Comment:
If you could change the name of this function to `book_to_nets` (also it's misspelled as compact, lol)